x-pack/plugins/observability/server/services/slo/summary_transform_generator/summary_transform_generator.ts

```diff
@@ -6,13 +6,14 @@
  */
 
 import { TransformPutTransformRequest } from '@elastic/elasticsearch/lib/api/types';
+import { DataViewsService } from '@kbn/data-views-plugin/common';
 import { SLO } from '../../../domain/models';
 import { generateSummaryTransformForOccurrences } from './generators/occurrences';
 import { generateSummaryTransformForTimeslicesAndRolling } from './generators/timeslices_rolling';
 import { generateSummaryTransformForTimeslicesAndCalendarAligned } from './generators/timeslices_calendar_aligned';
 
 export interface SummaryTransformGenerator {
-  generate(slo: SLO): TransformPutTransformRequest;
+  generate(slo: SLO, dataViewService: DataViewsService): TransformPutTransformRequest;
```

Comment:
I don't think we need the dataview service for generating the summary transform?